lib/src/components/NxThreatIndicator/NxThreatIndicator.tsx

```diff
@@ -21,9 +21,10 @@ const NxThreatIndicator: FunctionComponent<Props> =
           policyThreatLevel != null ? categoryByPolicyThreatLevel[policyThreatLevel] :
           'unspecified',
 
-        className = `nx-threat-indicator nx-threat-indicator--${category}`;
+        className = `nx-threat-indicator nx-threat-indicator--${category}`,
+        title = `${category} threat level indicator`;
 
-    return <NxFontAwesomeIcon className={className} icon={faCircle} />;
+    return <NxFontAwesomeIcon title={title} className={className} icon={faCircle}/>;
```

Comment:
Using `title` activates browser-native tooltips.  We want to avoid this because we have our own tooltips which look different.  If we actually want this text to be visible in a tooltip, we should use `NxTooltip`.  If we don't, we should use `aria-label`.

Reply:
85f72f39f457d112a10b8152d8b6f90b848fcfbd

Reply:
> Using title activates browser-native tooltips. We want to avoid this because we have our own tooltips which look different. If we actually want this text to be visible in a tooltip, we should use NxTooltip. If we don't, we should use aria-label.

After more investigation I want to elaborate a little bit.  It turns out that `FontAwesomeIcon` (which back `NxFontAwesomeIcon`) doesn't apply the `title` attribute natively, but rather turns it into an svg `<title>` element which it then sets up as the describedby on the svg.  This still has the effect of causing the browser to render a native tooltip, which we don't really want.  What else is interesting though is that the `<svg>` has a role of "img".  If you don't specify a title on it, FontAwesome sets it up with `aria-hidden` causing it not to be read at all.  If you add the `title` however, `aria-hidden` is not applied which causes it to be read out (in ChromeVox at least) as "severe threat level indicator. image."  The word "image" being in there doesn't seem ideal to me so I wonder if we can/should override that role.  I also wonder if VoiceOver does that too or just ChromeVox.

Reply:
VO also reads the aria label (or title) and then "image".
About the role, I'm not sure what should we use then, does role="icon" sound good?

Reply:
> does role="icon" sound good?

As far as I can tell that isn't a valid role.  Here's the list I've been using: https://www.w3.org/WAI/PF/aria/roles#role_definitions

Reply:
I don't see a role more appropriate than image. We could put role="", what do you think @rpokorny ?

Reply:
> I don't see a role more appropriate than image. We could put role="", what do you think @rpokorny ?

`role="none"` or `role="presentation"` seem fine to me, as long as they don't disable the label/title reading.  I'm not sure leaving it blank is valid.

Reply:
I used presentation, none did not sound right to me
cab3cd74528ef3b4266b9328b5a4ad6ef4a25d95